packages/ckeditor5-fullscreen/theme/fullscreen.css

```diff
@@ -44,25 +44,30 @@ Fullscreen layout:
 }
 
 .ck-fullscreen__main-container .ck-fullscreen__editor-wrapper {
+	--ck-fullscreen-editor-top-margin: 28px;
```

Comment:
Good idea with extracting it to the variable, we should probably do it with some other properties at some point.